src/components/Combobox.tsx

```diff
@@ -149,11 +152,23 @@ function Combobox<T>({
     }
   };
 
+  const createOption = () => {
+    if (!onCreate) return;
+
+    const optionValue = onCreate(inputValue);
+    selectOption(optionValue);
```

Comment:
It seemed to make sense to select the option on creation - interested to hear thoughts on this.